src/styles/blocks/card.scss

```diff
@@ -0,0 +1,103 @@
+.card {
+  display: flex;
+  box-sizing: border-box;
+  flex-direction: column;
+  align-items: center;
+  max-width: 200px;
+  margin: 0 24px 46px;
```

Comment:
You could't set margin to block, only for elements. Try to use mix, by adding `catalog__card` class to `card` block. And move margin there.